src/main/java/internet/shop/Factory.java

```diff
@@ -16,8 +16,27 @@
 import internet.shop.service.impl.ItemServiceImpl;
 import internet.shop.service.impl.OrderServiceImpl;
 import internet.shop.service.impl.UserServiceImpl;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import org.apache.log4j.Logger;
 
 public class Factory {
+
+    private static Logger logger = Logger.getLogger(Factory.class);
+    private static Connection connection;
+
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+            connection =
+                    DriverManager.getConnection("jdbc:mysql://localhost/store?"
+                            + "user=Taras&password=October2019&serverTimezone=UTC");
+        } catch (ClassNotFoundException | SQLException e) {
```

Comment:
never hard code credentials! Move driver class, jdbc url and credentials to property file

Reply:
done